exam001/ViewModels/DetailViewModel.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  DetailViewModel.swift
+//  exam001
+//
+//  Created by Seif Ghotouk on 11/27/18.
+//  Copyright © 2018 Adnan Al-Akhrass. All rights reserved.
+//
+
+import UIKit
+import Firebase
+
+class DetailViewModel: NSObject {
+    
+    public var ref : DatabaseReference!
```

Comment:
`ref` should be written as full type name
`public var databaseReference: DatabaseReference!`

check `code styling` for spaces before and after `, . : =`